content/en/tutorials/working-with-grpc-client.md

```diff
@@ -103,20 +103,20 @@ If the user loses a device _(unregistered / no more shared with the user)_, the
 
 ### Get Resource from Device
```

Comment:
This is only documentation of API. Please document the pending command concept as a new feature in features section.